docs/conf.py

```diff
@@ -18,9 +18,9 @@
 TAGS = []
 BRANCHES = ["master", "branch-5.1", "branch-5.2", "branch-5.4", "branch-6.0", "branch-6.1"]
```

Comment:
I assume that `branch-5.1` , `branch-5.2` and `branch-5.4` can be removed once `6.1` is out

Reply:
Most open source projects do keep old version docs, tagged as not supported.